### ISSUE TITLE
New user / apply crashes if no message areas defined

### ISSUE THREAD

**Issue by Fastidious** [NuSkooler/enigma-bbs#19]
Got it to run, using defaults. Getting this after using "Apply:"

```
(node) Buffer.get is deprecated. Use array indexes instead.
/Users/David/Desktop/enigma-bbs/mods/apply.js:105
				message_area_name	: getDefaultMessageArea().name,
				                 	                         ^

TypeError: Cannot read property 'name' of undefined
    at validationResult (/Users/David/Desktop/enigma-bbs/mods/apply.js:105:48)
    at userIdAndName (/Users/David/Desktop/enigma-bbs/mods/apply.js:62:4)
    at Statement.onResults (/Users/David/Desktop/enigma-bbs/core/user.js:399:6)
```

I am testing Enigma under OS X right now.

**Comment by NuSkooler** [comment]
@Fastidious, thanks for the report! This is a great example of the kind of things I miss since I'm used to my own workflow with this thing.

You should be able to get around this for now by adding at least one message area to your `config.hjson` file. For example:
```hjson
messages: {
  areas: [
    { name: "bug_reports", desc: "ENiGMA Bug Reports", groups: [ "users" ] }
  ]
}
```

Note that the above configuration block will change with #11. This is certainly a bug though, as the code needs to check for empty.

**Comment by NuSkooler** [comment]
Fixed with 3ad43c0. At startup the system will check for required additions to the config in `config.hjson`
